src/main/java/learnositysdk/request/Telemetry.java

```diff
@@ -0,0 +1,160 @@
+package learnositysdk.request;
+
+import org.apache.commons.io.IOUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.nio.charset.Charset;
+import java.util.Locale;
+import java.util.Properties;
+
+class Telemetry {
+	private static boolean telemetryEnabled = true;
+	private static JSONObject telemetryData = null;
+
+	// Java Runtimes (as expected in lib-metrics)
+	private final static String RT_OPEN_JDK = "openjdk";
+	private final static String RT_JAVA = "java";
+
+	// Operating Systems (as expected in lib-metrics)
+	private final static String OS_DARWIN = "darwin";
```

Comment:
Ditto for the OS, also what about runtimes and OSes we don't know about. Like, shava running on Minix, or whatever. We should have a fallback.

Reply:
Similar to above with the formatting expected in lib-metrics, _plus_ Java on Mac returns "Mac OS X" instead of "darwin" like we get from our other languages. So using these helped ensure they were the formats we expected. Other OSes are handled below in the getPlatform function.

Reply:
Makes sense. Java+OS X is a great recipe for pain...